system/config/templates/features/aca_shop_market/shop_benefits_2023.yml

```diff
@@ -138,3 +138,6 @@ registry:
       - key: :lowest_cost_silver_product_2023
         item: '94506DC0350033-01'
         is_enabled: <%= ENV['SHOP_OSSE_ELIGIBILITY_2023_IS_ENABLED'] || false %>
+      - key: :aca_shop_osse_audit_log_2023
+        item: :aca_shop_osse_audit_log_2023
+        is_enabled: <%= ENV['SHOP_OSSE_AUDIT_LOG_2023_IS_ENABLED'] || false %>
```

Comment:
is there any reason why we used the year on the feature flag? I think this a stand-alone feature that is not year dependent, but I may be not seeing something

Reply:
For OSSE, we have the feature enabled/disabled based on the year. I'm not sure if the same applies to OSSE Audit-Log. I can check with Will on this one. @polographer

Reply:
Confirmed with Will that it can just be one feature flag. Will update the code.